test/unit/calculators/statutory_sick_pay_calculator_test.rb

```diff
@@ -1071,7 +1073,7 @@ class StatutorySickPayCalculatorTest < ActiveSupport::TestCase
           start_date = Date.parse("16 April 2020") # thursday
           end_date = Date.parse("21 April 2020") # tuesday
 
-          should "pay SSP if they are shielding with GP letter" do
+          should "pay SSP for all days off from 16 April if they are shielding with GP letter" do
```

Comment:
This should be on the test above, rather than this one (I almost made the same mistake when I made the suggestion!). It is the one above that starts before 16th April and therefore the `days_paid` bit could be ambiguous.